src/Symfony/Component/Workflow/Tests/EventListener/GuardListenerTest.php

```diff
@@ -69,6 +69,16 @@ public function testWithSupportedEventAndAccept()
         $this->assertTrue($event->isBlocked());
     }
 
+    public function testWithNoTokenStorage()
+    {
+        $event = $this->createEvent();
+        $this->tokenStorage = null;
```

Comment:
Cool, thanks for your patience here.  First time contributing!

I'll get this done right away